app/components/cases/new-subcase-form.hbs

```diff
@@ -0,0 +1,191 @@
+    <Auk::Toolbar::Group>
+      <Auk::Toolbar::Item>
+        <AuLink
+          @skin="button-naked"
+          @icon="hierarchical-back"
+          @hideText={{true}}
+          {{on "click" this.transitionBack}}
+        >
+          {{t "back"}}
+        </AuLink>
+      </Auk::Toolbar::Item>
+      <Auk::Toolbar::Item>
+        <div class="auk-u-my">
+          <span class="auk-overline auk-u-block">
+            {{t "case"}}
+          </span>
+          <h4 class="auk-toolbar-complex__title">
+            <Auk::AbbreviatedText
+              @text={{or @decisionmakingFlow.case.shortTitle @decisionmakingFlow.case.title}}
+              @maxLength={{250}}
+            /> - {{t "create-subcase"}}
+          </h4>
+        </div>
+      </Auk::Toolbar::Item>
+    </Auk::Toolbar::Group>
+  </Auk::Toolbar>
+</Auk::Navbar>
+
+<div class="auk-scroll-wrapper">
+  <div class="auk-scroll-wrapper__body">
+    <div class="au-o-region">
+      <div class="au-o-layout">
+        <Auk::Panel class="auk-u-mb-2">
+          <Auk::Panel::Header
+            @size="auto"
+            class="au-u-background-gray-100"
+          >
+            <Auk::Toolbar @auto={{true}} as |Toolbar|>
+              <Toolbar.Group @position="left">
+                <h4 class="auk-toolbar__title">
+                  {{t "general-information"}}
+                </h4>
+              </Toolbar.Group>
+            </Auk::Toolbar>
+          </Auk::Panel::Header>
+          <Auk::Panel::Body>
+            <Utils::LoadableContent
+              @isLoading={{this.areLoadingTasksRunning}}
+              @reserveSpace={{true}}
+            >
+              <div class="au-c-form">
+                <div class="auk-o-flex auk-o-flex--justify-between auk-o-flex--vertical-center">
+                  <AuFormRow>
+                    <AuFieldset as |Fieldset|>
+                      <Fieldset.legend>{{t "subcase-type"}}</Fieldset.legend>
+                      <Fieldset.content>
+                        <AuRadioGroup
+                          @alignment="inline"
+                          @selected={{this.agendaItemType}}
+                          @onChange={{this.onChangeAgendaItemType}}
+                          as |Group|
+                        >
+                          {{#each this.agendaItemTypes as |type|}}
+                            <Group.Radio
+                              @value={{type}}
+                            >
+                              {{type.label}}
+                            </Group.Radio>
+                          {{/each}}
+                        </AuRadioGroup>
+                      </Fieldset.content>
+                    </AuFieldset>
+                  </AuFormRow>
+                  <AuFormRow>
+                  <AuButton
+                    @skin="secondary"
+                    @icon="copy-paste"
+                    {{on "click" (fn this.saveCase true)}}
+                    @disabled={{this.saveCase.isRunning}}
```

Comment:
`this.saveCase` is a function, not a task, so it will never have an `isRunning` property, so this disabled doesn't work. You can currently click it a bunch of times and it will create a bunch of empty subcases.

Reply:
Perhaps we should conditionally show this button. I don't think it makes a lot of sense to show "Kopieer voorgaande stap" when you're creating the first one.